test/k8s-integration/cluster.go

```diff
@@ -167,6 +167,10 @@ func clusterUpGKE(gceZone, gceRegion string, numNodes int, imageType string, use
 		cmdParams = append(cmdParams, "--enable-autorepair")
 	}
 
+	if isVariableSet(gkeNodeVersion) {
```

Comment:
nit: should there be validation against specifying both release channel and node version?

Reply:
I don't think that's an error. I just ran

gcloud beta container clusters create --release-channel rapid --node-version 1.16.8-gke.15 skewed

and it succeeded. It might be useful to test such skewed deployments (although it looks like one might need to be careful about auto-upgrade---maybe that should be automatically disabled whenever both release channel and node version are specified? Or maybe we should just disable auto-upgrade all the time anyway as we probably never want that to happen during a test?)

Reply:
It works if we specify release channel and node version. Master version picks the release version, while nodes pickup the node version.

Reply:
ah ok thanks for checking. Yeah in hindsight if this were not the case, the skew tests we have planned with node version = master version - 2 would be tough.

Disabling auto-upgrade all the time makes sense to me. Removes the possible confusion during the rare occasion when a short-lived test cluster is running while an auto-upgrade is being pushed out.

Reply:
I see error when using release channels, and --no-enable-autoupgrade option.
e.g.
```
Auto_upgrade cannot be false when release_channel REGULAR is set
```

Reply:
Ok, let's defer the autoupgrade fiddling until we understand the use cases a bit better. It may be that it's not a problem for tests, we'd have to dig into the details more.